client/signup/steps/woocommerce-install/transfer/install-plugins.tsx

```diff
@@ -9,35 +9,82 @@ import {
 import { getAtomicSoftwareStatus } from 'calypso/state/atomic/software/selectors';
 import { getSiteWooCommerceUrl } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import Error from './error';
 import Progress from './progress';
 import './style.scss';
 
-export default function InstallPlugins(): ReactElement | null {
+// Timeout limit for the install to complete.
+const TIMEOUT_LIMIT = 1000 * 15; // 15 seconds.
```

Comment:
Guessing this was for testing, transfers take 40-50 seconds at times.

```suggestion
const TIMEOUT_LIMIT = 1000 * 60; // 60 seconds.
```

Reply:
:facepalm: this is the install flow :smile: